### PR TITLE
ci: Allow only ts files for new cypress tests

### PR DESCRIPTION
**Pull request by sarojsarab**
## Description
 - Added steps in client-build.yml to check the newly added files under cypress/e2e
 - Commenting in the PR with the file names in case of files written in js
 - Failing the workflow in the above case
#### Type of change
 - client-build.yml changes
#### How Has This Been Tested?
- Workflow run

### CODE REVIEW

File: .github/workflows/client-build.yml [hunk 1]
```diff
@@ -61,6 +61,40 @@ jobs:
...
+          echo "non_ts_files_count=${#non_ts_files[@]}" >> $GITHUB_OUTPUT
+          
+      # Comment in PR if test files are not written in ts and fail the workflow    
+      - name: Comment in PR if test files are not written in ts
+        if: steps.check_files.outputs.non_ts_files_count != 0
+        uses: peter-evans/create-or-update-comment@v1
+        with:
+          issue-number: ${{ github.event.client_payload.pull_request.number }}
+          body: |
+            <b>Below files are written in js expected ts. Please fix and rerun the workflow.</b>
```
Comment (Aishwarya-U-R) [inline .github/workflows/client-build.yml:93]: Can it like below:
```
Below files are written in js 🔴
Expected format ts. Please fix and retrigger ok-to-test.
```

File: .github/workflows/client-build.yml [hunk 2]
```diff
@@ -61,6 +61,41 @@ jobs:
...
+          for file in "${files[@]}"; do
+            if [[ $file != *.ts ]]; then
+              non_ts_files+=("<li>$file")
+            fi
+          done
+          echo "non_ts_files=${non_ts_files[@]}" >> $GITHUB_OUTPUT
+          echo "non_ts_files_count=${#non_ts_files[@]}" >> $GITHUB_OUTPUT
+          
+      # Comment in PR if test files are not written in ts and fail the workflow    
+      - name: Comment in PR if test files are not written in ts
```
Comment (Aishwarya-U-R) [inline .github/workflows/client-build.yml:87]: It will comment & stop client-build further right?

**Comment by Aishwarya-U-R** [comment]
@yatinappsmith - Should we merge this this in now? Or should we alert people this is coming? Or alert after merged :)

**Comment by yatinappsmith** [comment]
> 

Initial check should be only to prevent addition of new files.

**Comment by Aishwarya-U-R** [comment]
Yes, this PR achieves only that!

File: .github/workflows/client-build.yml (hunk 2 above)
Comment (sarojsarab) [reply .github/workflows/client-build.yml:87]: Yes, it comment and fail the workflow

**Review by Aishwarya-U-R — APPROVED** [review]

**Pull request merged**